compiler/include/symbol.h

```diff
@@ -353,10 +353,16 @@ class FnSymbol : public Symbol {
   int codegenUniqueNum;
   const char *doc;
 
+  // The following fields are used only when hasFlag(FLAG_PARTIAL_COPY)
```

Comment:
Can you please expand on the intro to the set of member variables.  It doesn't need to be long discussion
but perhaps a few sentences for the (1) the reader who isn't particularly familiar with the implications of
FLAG_PARTIAL_COPY (2) and some sort of remark to the effect that these fields suggest that the class
should be refactored.